sopt/src/main/scala/dagr/sopt/OptionLookup.scala

```diff
@@ -149,11 +149,14 @@ trait OptionLookup {
     */
   private[sopt] def findExactOrPrefix(optionName: String): List[OptionAndValues] = {
     // first see if the name is just in the map
-    if (optionMap.contains(optionName)) return List(optionMap.get(optionName).get)
-    // next, check abbreviations
-    getOptionNamesWithPrefix(optionName)
-      .map { name => optionMap.get(name).get }
-      .toList
+    if (optionMap.contains(optionName)) {
```

Comment:
It's up to you, but I find that if each branch is a single expression, that it also looks fine without the braces.